magic.py

```diff
@@ -1,13 +1,13 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # coding: utf-8
 
 from datetime import datetime
 import argparse
-import HTMLParser
+import html.parser
 import json
 import os
 import sys
-import urllib2
+import urllib.request, urllib.error, urllib.parse
```

Comment:
can we just import `urllib`, otherwise have one import per line

Reply:
I tried, it raises  
`AttributeError: module 'urllib' has no attribute 'request'`